File: DAL/Models/Secteurs.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL.Models
{
   public class Secteurs: AuditableEntity
    {
        public Secteurs()
        {
            this.SecteursFournisseurs = new HashSet<SecteursFournisseurs>();
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public string description { get; set; }

        //navigation

        public ICollection<SecteursFournisseurs> SecteursFournisseurs { get; set; }
    }
}

File: Atomics Manager/ClientApp/src/app/admin/models/secteurs.ts
export class Secteurs {
    constructor(name?: string, description?: string, fournisseurs?: string[]) {
        this.name = name;
        this.description = description;
        this.fournisseurs = fournisseurs;
    }

    public id: number;
    public name: string;
    public description: string;

    public fournisseurs: string[];


}
